sopel_modules/rat_board.py

```diff
@@ -866,18 +866,21 @@ def func_clear(bot, trigger, rescue, markingForDeletion=False, *firstlimpet):
     if len(firstlimpet) == 1:
         rat = getRatId(bot, firstlimpet[0], rescue.platform)['id']
         if rat != "0":
-            rescue.firstLimpet = rat
-            dt = datetime.date(2017, 4, 1)
-            if datetime.date.today() == dt:
-                bot.say(
-                    'Your case got closed and you fired the First Limpet! Check if the paperwork is correct here: http://t.fuelr.at/a41',
-                    firstlimpet[0])
-            else:
-                bot.say(
-                'Your case got closed and you fired the First Limpet! Check if the paperwork is correct here: ' + url,
-                firstlimpet[0])
             if rat not in rescue.rats:
+                try:
+                    callapi(bot, 'PUT', '/rescues/assign/' + str(rescue.id), data={'data':[rat]}, triggernick=str(trigger.nick))
+                    # debug. disable for release
+                    bot.reply('Rat was not assigned to case. Now it is! You\'re welcome!')
+                except ratlib.api.http.APIError:
```

Comment:
```suggestion
                except ratlib.api.http.APIError:
                logging.exception()
```
Do **not** just silently suppress an exception please :(

Reply:
we're not. look at the source code, there's a bot.reply that just isn't being shown.

Reply:
this except block catches `ratlib.api.http.APIError` exceptions, im not seeing anything under this except block that is outputting a stacktrace. 
Nothing in containing scopes is going to catch this exception unless you re-raise it.

Reply:
see IRC.